.env

```
@@ -0,0 +1,5 @@
+COMPOSE_PATH_SEPARATOR=; 
+COMPOSE_FILES=docker-compose.yml;docker/dev.yml
```

There is a typo here @Aju100 and that's what makes it not working !
```suggestion
COMPOSE_FILE=docker-compose.yml;docker/dev.yml
```